tests/opwen_email_server/utils/test_email_parser.py

```diff
@@ -80,3 +80,13 @@ def test_gets_domains_with_cc_and_bcc(self):
         domains = email_parser.get_domains(email)
 
         self.assertSetEqual(domains, {'bar.com', 'com'})
+
+
+class ConvertImgUrlToBase64(TestCase):
+    def test_inlines_image(self):
```

Comment:
Please also add test cases for:

- [ ] What happens if the image is surrounded by other HTML content (can just augment this test)?
- [ ] What happens if the image tag does not have a src attribute? (The img tag should not be changed.)
- [ ] What happens if the image URL can't be fetched? (The img tag should not be changed.)